### PR TITLE
Updated Main.kt

### PR DESCRIPTION
**Pull request by MarcoChrist**
@nicolaiparlog This PR includes the updated Main.kt with the traffic.csv file, the FileAccess.kt so that reader and writer work properly.
The results.csv which shows whether the cars are able to drive due to trafficjam is also included.

If im not mistaken, this should solve the Issues #28 #29 #30 

Also the other two PR created by NikMass should work fine when all this comes together.

### CODE REVIEW

File: src/main/Resources/traffic.csv [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Car, WantsToDrive
```
Comment (nipafx) [inline src/main/Resources/traffic.csv:1]: The folder should be called `resources`, not `Resources` (capitalization matters).

File: src/main/kotlin/FinalProject/Main.kt [hunk 2]
```diff
@@ -1,5 +1,93 @@
...
+	val rn = RoadNetwork(7, Cars)
+	rn.simulationStep()
+	rn.printCarsJammed()
+}
+
 fun main(args: Array<String>) {
-    println("Hello, World!")
+	val Cars : MutableList<Car> = parseCSV()
+
+	scenario(Cars)
```
Comment (nipafx) [inline src/main/kotlin/FinalProject/Main.kt]: The `scenario` function is meant to execute an entire simulation (i.e. create cars, simulate, write results) without any CSV input/output. The reason is that you should always have one way to run the simulation _without_ having to rely on CSV parsing (which is easy to break accidentally).

So please change `scenario` to create its own cars and write a second function (maybe `csvScenario`) that loads from CSV, simulates, and writes to CSV. `main` should then execute one or both of these functions.

File: src/main/kotlin/FinalProject/Main.kt [hunk 3]
```diff
@@ -1,5 +1,93 @@
...
+
+
+	for (record in allRows){
+		val CarIdStr : String = record.values[0]
+		val CarId : Int = CarIdStr.toInt()
+
+		val WantDriveStr : String = record.values[1]
+		val WantDrive : Boolean = WantDriveStr.toBoolean()
+
+		CarListCSV.add(Car(d = WantDrive))
```
Comment (nipafx) [inline src/main/kotlin/FinalProject/Main.kt]: Aren't you going to use the `CarId`?

File: src/main/kotlin/FinalProject/Main.kt [hunk 4]
```diff
@@ -1,5 +1,93 @@
...
+
+	printResultsToCSV(Cars)
+
+
+
 }
+
+
+fun parseCSV(): MutableList<Car> {
+	val CarListCSV: MutableList<Car> = mutableListOf()
```
Comment (nipafx) [inline src/main/kotlin/FinalProject/Main.kt]: Please start all variable names with lower letters.

**Review by nipafx — CHANGES_REQUESTED** [review]
Overall this looks very good. But Travis is complaining and you have to fix that.

:bulb: Before any commit you hould make sure that `mvn clean install` does not result in errors.

:bulb: Smaller pull requests that address less issues make it easier to review them.

File: src/main/Resources/traffic.csv (hunk 1 above)
Comment (MarcoChrist) [reply src/main/Resources/traffic.csv:1]: @nicolaiparlog why has this to be lower case? For linux it matters, but why should it be all lower case?

File: src/main/kotlin/FinalProject/Main.kt (hunk 2 above)
Comment (MarcoChrist) [inline src/main/kotlin/FinalProject/Main.kt]: @nicolaiparlog 
The scenario function has a parameter which takes the car list, that means the scenario function itself is independent of CSV. The main function handles the CSV reading/writing

File: src/main/Resources/traffic.csv (hunk 1 above)
Comment (nipafx) [reply src/main/Resources/traffic.csv:1]: I don't know whether there was a particular reason to use lower case back in the old days. I just know it as a convention. A consequence of that is that Maven uses certain paths (like `src/main/resources`) by default and you'd have to actually configure it to use different values.

**Pull request merged**